addons/website/static/src/snippets/s_process_steps/000.scss

```diff
@@ -18,33 +60,90 @@
                 }
-            }
-            .s_process_step_icon {
-                span:after {
-                    width: 100%;
+                span.s_process_step_curved_arrow {
+                    @include base_arrow;
+                    border-radius: 50%/40% 50% 0 0;
+                    border-width: 4px;
+                    top: -15%;
+                    &:after {
+                        top: auto;
+                        right: -0.75%;
+                        border-width: 20px 7px 0px 7px;
+                        transform: rotate(-60deg);
+                    }
                 }
             }
-            &:first-child .s_process_step_icon,
-            &:last-child .s_process_step_icon {
-                span:after {
-                    width: 50%;
+            &:nth-child(even) .s_process_step_icon {
+                span.s_process_step_curved_arrow {
+                    top: 70%;
+                    transform: rotate(180deg);
+                    &:after {
+                        left: -1.8%;
+                        top: -5%;
+                        transform: rotate(62deg);
+                    }
                 }
             }
             &:first-child .s_process_step_icon {
-                .fa:after { right: 0; }
-                .fa.float-right:after { width: 0; }
+                span.s_process_step_line_left {
+                    width: 0;
+                }
             }
             &:last-child .s_process_step_icon {
-                span:after { left: 0; }
-                .fa {
-                    &:after { left: 0; }
-                    &.float-left:after { width: 0; }
+                span.s_process_step_line_right {
+                    width: 0 !important;
+                    &:after {
+                        border-width: 0 !important;
+                    }
+                }
+            }
+            &.col-lg-6 {
+                @include override_curved_arrow($borderWidth: 6px);
+            }
+            &.col-lg-2,
+            &.col-lg-3 {
+                span.s_process_step_arrow {
+                    right: -18%;
+                    width: 40%;
+                }
+            }
+            &.col-lg-2 {
+                @include override_curved_arrow($angle: -50deg, $pointerRight: -2.5%, $pointerTop: -3.5%);
+                &:nth-child(even) .s_process_step_icon {
+                    @include override_curved_arrow($angle: 55deg, $pointerTop: -8%, $pointerLeft: -3%);
+                }
+            }
+        }
+        .o_container_small .s_process_step {
+            &.col-lg-2 {
+                span.s_process_step_arrow {
+                    right: -8%;
+                }
+            }
+            &.col-lg-3 {
+                &:nth-child(odd) .s_process_step_icon {
+                    @include override_curved_arrow($angle: -52deg, $pointerTop: -1.5%);
+                }
+            }
+        }
+    }
+    @include media-breakpoint-up(xl) {
```

Comment:
I doubt that media breakpoint rules are necessary for this either: this must be generic as "arrows *between* columns", the column size or the screen size should not impact anything :wink: